### PR TITLE
93x - Present Calo conditions used at p5, for v2 L1 menu and post-MD2.

### PR DESCRIPTION
**Pull request by rekovic**
pr93x

Calo Conditions configuration to reflect what currently used at P5.
It only contains 3 configuration files. 

Change w.r.t. currently in CMSSW Calo Conditions 2017_v1_8 :
- invoke 2nd EG Iso WP
- update Layer1 HFSF for the first 12 bins (from flat 0.7 to flat 1.0)
- MET PUS v6, changed from v4

To test it needs external PR https://github.com/cms-data/L1Trigger-L1TCalorimeter/pull/21

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by rekovic** [comment]
Reason of failing tests is: New LUT used in this PR is not found in the search path.
Will try test with with explicite external dist.

**Comment by rekovic** [comment]
please test with cms-sw/cmsdist#3316

**Comment by arunhep** [comment]
@rekovic as discussed in morning with @franzoni please cherry pick this commit https://github.com/franzoni/cmssw/commit/4a1e1c8c2a431344bfe85f1bdfeb0d55095ac652 this contains the GT updates for 2017 MC in order to remove 0.7 factor from HF LUTs

**Comment by rekovic** [comment]
@arunhep Thx.

**Comment by rekovic** [comment]
please test with cms-sw/cmsdist#3316

**Comment by arunhep** [comment]
@rekovic here is the commit with including the latest JECs for HLT : https://github.com/franzoni/cmssw/commit/9844cd262e9e375144548eb0757ff04de0533071
@franzoni

**Comment by rekovic** [comment]
From private conversation with @franzoni, I am expecting another update from AlCa (~1-2 hrs) which I will cherry-pick to this PR.

**Comment by rekovic** [comment]
@arunhep 
Ah, OK, already available. Thx.

**Comment by rekovic** [comment]
please test with cms-sw/cmsdist#3316

**Comment by rekovic** [comment]
+1

**Comment by franzoni** [comment]
+1
@arunhep

File: L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
+#
```
Comment (davidlange6) [inline L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py]: hi @rekovic - all of these caloStage2Params* files are really still relevant? (we do have a code versioning system after all..)

**Comment by davidlange6** [comment]
otherwise, @arunhep -please paste in the GT changes summary so we know that its here were the hcalLUTs were updated (again?...)

**Comment by arunhep** [comment]
> otherwise, @arunhep -please paste in the GT changes summary so we know that its here were the hcalLUTs were updated (again?...)

@davidlange6 sure will do it

**Comment by arunhep** [comment]
# Summary of changes in Global Tags 
 
## Upgrade 
 
   * **PhaseI 2017 cosmics peak scenario** : [92X_upgrade2017cosmics_realistic_peak_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017cosmics_realistic_peak_v10) as [92X_upgrade2017cosmics_realistic_peak_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017cosmics_realistic_peak_v8) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_upgrade2017cosmics_realistic_peak_v10/92X_upgrade2017cosmics_realistic_peak_v8): 
      * remove of HF  TP scale factor 0.7 from HCALLUTs : https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2977.html
      * Update of JECs@HLT to the ones currently running for data-taking: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/3091.html

   * **PhaseI 2017 cosmics scenario** : [92X_upgrade2017cosmics_realistic_deco_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017cosmics_realistic_deco_v10) as [92X_upgrade2017cosmics_realistic_deco_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017cosmics_realistic_deco_v8) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_upgrade2017cosmics_realistic_deco_v10/92X_upgrade2017cosmics_realistic_deco_v8): 
      * remove of HF  TP scale factor 0.7 from HCALLUTs : https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2977.html
      * Update of JECs@HLT to the ones currently running for data-taking: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/3091.html

   * **PhaseI 2017 design scenario** : [92X_upgrade2017_design_IdealBS_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017_design_IdealBS_v10) as [92X_upgrade2017_design_IdealBS_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017_design_IdealBS_v8) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_upgrade2017_design_IdealBS_v10/92X_upgrade2017_design_IdealBS_v8): 
      * remove of HF  TP scale factor 0.7 from HCALLUTs : https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2977.html
      * Update of JECs@HLT to the ones currently running for data-taking: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/3091.html

   * **PhaseI 2017 realistic scenario** : [92X_upgrade2017_realistic_v10](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017_realistic_v10) as [92X_upgrade2017_realistic_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/92X_upgrade2017_realistic_v8) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_upgrade2017_realistic_v10/92X_upgrade2017_realistic_v8): 
      * remove of HF  TP scale factor 0.7 from HCALLUTs : https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2977.html
      * Update of JECs@HLT to the ones currently running for data-taking: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/3091.html

File: L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py (hunk 1 above)
Comment (rekovic) [inline L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py]: hi @davidlange6 
These files provide different modes of configuration of L1T Calo emulation. 
Apart from the default configuration, one can configure the emulator in any of the available modes via customisation functions.

**Comment by davidlange6** [comment]
Sure - but they are getting rather numerous and repetitive,no?

**Comment by davidlange6** [comment]
ping @rekovic

**Comment by davidlange6** [comment]
merge
questions to be understood before back port

**Comment by Dr15Jones** [comment]
I believe this pull request is responsible for the many failures in last nights RelVals.  E.g. 
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc630/CMSSW_9_3_X_2017-07-31-1100/pyRelValMatrixLogs/run/134.813_RunCosmics2015C+RunCosmics2015C+RECOCOSDRUN2+ALCACOSDRUN2+HARVESTDCRUN2/step2_RunCosmics2015C+RunCosmics2015C+RECOCOSDRUN2+ALCACOSDRUN2+HARVESTDCRUN2.log

The error is
```
----- Begin Fatal Exception 31-Jul-2017 13:32:06 CEST-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named step2_RAW2DIGI_L1Reco_RECO_DQM_ALCA.py
Exception Message:
python encountered the error: <type 'exceptions.RuntimeError'>
edm::FileInPath unable to find file L1Trigger/L1TCalorimeter/data/lut_towEtThresh_2017v6.txt anywhere in the search path.
The search path is defined by: CMSSW_SEARCH_PATH
${CMSSW_SEARCH_PATH} is: /build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_3_X_2017-07-31-1100/src:/build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_3_X_2017-07-31-1100/external/slc7_amd64_gcc630/data:/cvmfs/cms-ib.cern.ch/nweek-02483/slc7_amd64_gcc630/cms/cmssw-patch/CMSSW_9_3_X_2017-07-31-1100/src:/cvmfs/cms-ib.cern.ch/nweek-02483/slc7_amd64_gcc630/cms/cmssw-patch/CMSSW_9_3_X_2017-07-31-1100/external/slc7_amd64_gcc630/data
Current directory is: /build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_3_X_2017-07-31-1100/pyRelval/134.813_RunCosmics2015C+RunCosmics2015C+RECOCOSDRUN2+ALCACOSDRUN2+HARVESTDCRUN2

----- End Fatal Exception -------------------------------------------------
```

**Comment by smuzaffar** [comment]
yes @Dr15Jones , it needed an external to go with it cms-sw/cmsdist#3316 .
I have merged the external and restarted 12h00 IB. 11h00 IB has been marked as failed and will not be used for any PR tests.

**Comment by davidlange6** [comment]
ah - because i didn't merge the external.. thanks to @smuzaffar for fixing already

**Pull request merged**